cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -64,38 +56,36 @@ CREATE OR REPLACE USER name
 | Create a new user, or if a user with the same name exists, replace it.
 | <<administration-security-administration-dbms-privileges-user-management, CREATE USER>> and
 <<administration-security-administration-dbms-privileges-user-management, DROP USER>>
-| `+`
-| `+`
 
-| [source, cypher]
+| [source]
 ----
 ALTER USER name SET {
 [PLAINTEXT \| ENCRYPTED] PASSWORD password
             [[SET PASSWORD] CHANGE [NOT] REQUIRED]
```

Comment:
These are very long spaces before the inner parts, they don't seem to be that far in on for example the create command

Reply:
I've made them the same, although afaict they were always like this.

Reply:
I know in some place the `[PLAINTEXT \| ENCRYPTED] PASSWORD password` were indented halfway and then the big gap made more sense, without that half step I don't think it does :P